src/components/Navbar/MobileSidebar/Layout/index.tsx

```diff
@@ -25,6 +28,9 @@ export default function NavbarMobileSidebarLayout({
   const data = useAllDocsData();
   const { versions } = data.default;
   const reversed = [...versions].reverse();
+  const items: NavbarNavLinkProps[] = useThemeConfig().navbar.items.filter(
+    (item) => item.label !== undefined
+  );
```

Comment:
Let's split this logic into a couple steps. It's a bit too much logic to have it all in one line